docs/content/usage/windows-machine-images.md

```diff
@@ -0,0 +1,383 @@
+## Overview
+
+Wakame has functionality
+specifically for supporting Windows, both for creating Windows seed image and for running and using multiple Windows instances in a Wakame environment. The use of Windows on Wakame-vdc is significantly different from the use of Linux instances.
```

Comment:
`for creating Windows seed image` is a grammar mistake. Either say *"a Windows seed image"* or *"Windows seed images"*.